### PR TITLE
USAGOV-1756-content-guide: Add wizard content manager guide

### PR DESCRIPTION
**Pull request by mikeredman**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1756

## Description

Add a markdown wizard content management guide to describe the fields and show how to create a wizard.

## Type of Changes

- [ ] New Feature
- [ ] Bugfix
- [ ] Frontend (Twig, Sass, JS)
  - Add screenshot showing what it should look like
- [ ] Drupal Config (requires "drush cim")
- [ ] New Modules (requires rebuild)
- [x] Documentation
- [ ] Infrastructure
  - [ ] CMS
  - [ ] WAF
  - [ ] Egress
  - [ ] Tools
- [ ] Other

## Testing Instructions

No testing

### Change Requirements

- [ ] Requires New Documentation (Link: {})
- [ ] Requires New Config
- [ ] Requires New Content

### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review

- [ ] Adds/updates software (including a library or Drupal module)
- [ ] Communication with external service
- [ ] Changes permissions or workflow
- [ ] Requires SSPP updates

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.

### CODE REVIEW

File: docs/Wizards_Content.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## How to create a wizard
+
+In order to create a new wizard, go to the Wizard Manager and press the button "Add term". This step will be used repeatedly in order to create a new wizard and new sub-terms and options of that header wizard. 
+
+On the Add term page, you will be asked and need to fill out the following information:
+
+* Name: Drupal's name for the taxonomy term
+* Language: Language
+* Option Name: This is the name that will show up when you are using the hierarchical tree-view within the Wizard Manager interface
```
Comment (mdranove) [inline docs/Wizards_Content.md]: Actually this field determines what to display in the options list on the term which leads to the term you are creating.

File: docs/Wizards_Content.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+* Description: This field is for additional text that you want displayed below the option in the option list. 
+* Heading/Question: This is the heading that will display at the top of your wizard question, in large red text. 
+* Language Toggle: Sets a translation for the current page and the translation goes into the language dropdown in the top right
```
Comment (mdranove) [inline docs/Wizards_Content.md]: should be "translation is linked to in the language switch link in the top right"

File: docs/Wizards_Content.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+* Intro: This field is a WYSIWYG text for below the Header/Question but above the options list and Body.
```
Comment (mdranove) [inline docs/Wizards_Content.md]: Should also be listed as "Optional"

File: docs/Wizards_Content.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+* Body: This field is for text that you want displayed above the options list and below the introduction. Optional
+* Footer: This field is for text that you want displayed below the options list. Optional
+* Header HTML: This is optional HTML field for content that will go at the top of the page, above all other things on the page. This is where analytics team can put in gtags and other scripts.
```
Comment (mdranove) [inline docs/Wizards_Content.md]: Specifically this is just for `<scripts>` that go in the `<head>`. Content managers should not use unless they know what they're doing.

File: docs/Wizards_Content.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+* Relations: This drop down is completely optional but allows you to set the parent of the term you are creating at initial creation. This is optional as the benefit of the hierarchical tree view is that after you create some of these terms for your wizard, you will be able to drag and drop them into the specified parent, as well as specific order so that you do not need to manually set the parent or weight.
+* Published: Do you want this wizard published or not
+* URL Alias: Specify an alternative path which the data can be accessed.
```
Comment (mdranove) [inline docs/Wizards_Content.md]: Specify the path at which the page can be accessed, e.g "/learn-where-report-scam"

File: docs/Wizards_Content.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+* Body: This field is for text that you want displayed above the options list and below the introduction. Optional
+* Footer: This field is for text that you want displayed below the options list. Optional
+* Header HTML: This is optional HTML field for content that will go at the top of the page, above all other things on the page. This is where analytics team can put in gtags and other scripts.
+* Relations: This drop down is completely optional but allows you to set the parent of the term you are creating at initial creation. This is optional as the benefit of the hierarchical tree view is that after you create some of these terms for your wizard, you will be able to drag and drop them into the specified parent, as well as specific order so that you do not need to manually set the parent or weight.
+* Published: Do you want this wizard published or not
+* URL Alias: Specify an alternative path which the data can be accessed. 
+
+## How to use the hierarchical tree view
+
+From the Wizard Manager, you can collapse, uncollapse, drag & drop, reorder and edit taxonomy terms from a single page.
```
Comment (mdranove) [inline docs/Wizards_Content.md]: And search by name

File: docs/Wizards_Content.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+* Footer: This field is for text that you want displayed below the options list. Optional
+* Header HTML: This is optional HTML field for content that will go at the top of the page, above all other things on the page. This is where analytics team can put in gtags and other scripts.
+* Relations: This drop down is completely optional but allows you to set the parent of the term you are creating at initial creation. This is optional as the benefit of the hierarchical tree view is that after you create some of these terms for your wizard, you will be able to drag and drop them into the specified parent, as well as specific order so that you do not need to manually set the parent or weight.
+* Published: Do you want this wizard published or not
+* URL Alias: Specify an alternative path which the data can be accessed. 
+
+## How to use the hierarchical tree view
+
+From the Wizard Manager, you can collapse, uncollapse, drag & drop, reorder and edit taxonomy terms from a single page. 
+
```
Comment (mdranove) [inline docs/Wizards_Content.md:36]: Need some info on how to add in the left hand nav menu

**Review by akf — DISMISSED** [review]

File: docs/Wizards_Content.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+* Name: Drupal's name for the taxonomy term
+* Language: Language
+* Option Name: This field is for naming the option within the options list for the term you are creating. 
+* Description: This field is for additional text that you want displayed below the option in the option list. 
+* Heading/Question: This is the heading that will display at the top of your wizard question, in large red text. 
+* Language Toggle: Sets a translation for the current page and the translation is linked to in the language switch link in the top right.
+* Intro: This field is a WYSIWYG text for below the Header/Question but above the options list and Body. Optional.
+* Body: This field is for text that you want displayed above the options list and below the introduction. Optional
+* Footer: This field is for text that you want displayed below the options list. Optional
+* Header HTML: This is optional HTML field for content that will go at the top of the page, above all other things on the page. This is just for <scripts> that go in the <head>. Content managers should not use unless they know what they are doing.
```
Comment (akf) [inline docs/Wizards_Content.md]: The tags here aren't rendering -- check out https://github.com/usagov/usagov-2021/blob/USAGOV-1756-content-guide/docs/Wizards_Content.md

I think backticks around these will fix it.

**Review by akf — COMMENTED** [review]
I approved this, but then I decided to look at the rendered markup and caught an issue -- see the comment.

**Review by akf — APPROVED** [review]

**Pull request merged**